docs/dev/install-docker.rst

```diff
@@ -65,7 +71,8 @@ initialize submodules with ``git submodule update --init --recursive``.
    This will ask you to create a superuser, and then will update your Firefox
    account settings.
 
-At that point, you're good to go. Access the webapp via this URL: http://localhost:8000/
+At that point, you're good to go. Access the webapp via this URL: http://localhost:8000/ or the custom SITE_URL.
```

Comment:
Note that while accessing via the custom `SITE_URL` seems to work on the surface, I'm experiencing some CSP/CORS issues due to the client-side code and HTML markup still referencing `localhost` here and there.

Reply:
@julen @adngdb That's probably caused due to the fact that our initial migration for pontoon-intro always sets base url to `http://localhost:8000/`. We could parametrize that or change after migrations are applied.